src/cmd/flux-terminus.c

```diff
@@ -372,7 +372,7 @@ static int start_service_daemon (flux_t *orig_h, optparse_t *p)
     }
```

Comment:
Commit message only mentions waitpid + EINTR, but commit also fixes a couple of short writes